File: Desafios em C#/2686 - A Mudança Continua!!.cs
using System;

//Developed by: @LucasMarcuzo

namespace _2685_A_Mudanca_Continua
{
    class Program
    {
        static void Main(string[] args)
        {
    	    float x;
            string t;

            while (!string.IsNullOrEmpty(t = Console.ReadLine())){

                double y = Convert.ToDouble(t);
        
                x = (int)y;
                int horas = ((int)(y * 240) / 3600) % 60;
                int minutos = ((int)(y * 240) / 60) % 60;
                int segundos = (int)(y * 240) % 60;
        
                if (x >= 0 && x < 90 || x == 360)
                    Console.WriteLine("Bom Dia!!\n{0:D2}:{1:D2}:{2:D2}", (horas + 6) % 24, minutos, segundos);
                else if (x >= 90 && x < 180)
                    Console.WriteLine("Boa Tarde!!\n{0:D2}:{1:D2}:{2:D2}", (horas + 6) % 24, minutos, segundos);
                else if (x >= 180 && x < 270)
                    Console.WriteLine("Boa Noite!!\n{0:D2}:{1:D2}:{2:D2}", (horas + 6) % 24, minutos, segundos);
                else if (x >= 270 && x < 360)
                    Console.WriteLine("De Madrugada!!\n{0:D2}:{1:D2}:{2:D2}", (horas + 6) % 24, minutos, segundos);
                    
            
            }
        }
    }
}

File: Desafios em JavaScript/2686 - A Mudança Continua!!.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var x;

while (lines.length > 0) {

    var y = parseFloat(lines.shift());

    x = parseInt(y);
    var horas = (parseInt((y * 240) / 3600) % 60);
    var minutos = (parseInt((y * 240) / 60) % 60);
    var segundos = parseInt((y * 240) % 60);

    let hora = ((horas + 6) % 24);
    let saldacao;

    if (x >= 0 && x < 90 || x == 360)
    {
        saldacao = "Bom Dia!!";
    }
    else if (x >= 90 && x < 180)
    {
        saldacao = "Boa Tarde!!";
    }
    else if (x >= 180 && x < 270)
    {
        saldacao = "Boa Noite!!";
    }
    else if (x >= 270 && x < 360)
    {
        saldacao = "De Madrugada!!";
    }

    console.log(`${saldacao}\n${hora.toString().padStart(2, '0')}:${minutos.toString().padStart(2, '0')}:${segundos.toString().padStart(2, '0')}`);
}